#define SPDLOG_LEVEL_NAMES with external FMT doesn't compile, starting from libfmt11.

This came up macOS/apple clang:

```bash
In file included from /opt/local/include/spdlog/common.h:410:
/opt/local/include/spdlog/common-inl.h:19:26: error: constexpr variable 'level_string_views' must be initialized by a constant expression
    static string_view_t level_string_views[] SPDLOG_LEVEL_NAMES;
                         ^                    ~~~~~~~~~~~~~~~~~~
/opt/local/include/spdlog/common-inl.h:19:47: note: non-constexpr constructor 'basic_string_view' cannot be used in a constant expression
    static string_view_t level_string_views[] SPDLOG_LEVEL_NAMES;
                                              ^
/Users/Koji/performous/game/log.hh:11:30: note: expanded from macro 'SPDLOG_LEVEL_NAMES'
#define SPDLOG_LEVEL_NAMES { "TRACE", "DEBUG", "INFO", "NOTICE", "WARNING", "ERROR", "OFF" }
                             ^
/opt/local/include/libfmt11/fmt/base.h:536:3: note: declared here
  basic_string_view(const Char* s)
  ^
```
Removing the `constexpr` from `static string_view_t level_string_views[] SPDLOG_LEVEL_NAMES;` makes it compile, but I don't think this is really the correct fix?

--- a/include/spdlog/tweakme.h
+++ b/include/spdlog/tweakme.h
@@ -104,6 +104,13 @@
 //
 // #define SPDLOG_LEVEL_NAMES { "MY TRACE", "MY DEBUG", "MY INFO", "MY WARNING", "MY ERROR", "MY
 // CRITICAL", "OFF" }
+//
+// Starting with C++17 use string_view_literals:
+//
+// #include <string_view>
+// using namespace std::string_view_literals;
+// #define SPDLOG_LEVEL_NAMES { "MY TRACE"sv, "MY DEBUG"sv, "MY INFO"sv, "MY WARNING"sv, "MY ERROR"sv, "MY
+// CRITICAL"sv, "OFF"sv }
 ///////////////////////////////////////////////////////////////////////////////
 
 ///////////////////////////////////////////////////////////////////////////////

--- a/include/spdlog/tweakme.h
+++ b/include/spdlog/tweakme.h
@@ -105,7 +105,7 @@
 // #define SPDLOG_LEVEL_NAMES { "MY TRACE", "MY DEBUG", "MY INFO", "MY WARNING", "MY ERROR", "MY
 // CRITICAL", "OFF" }
 //
-// Starting with C++17 use string_view_literals:
+// For C++17 use string_view_literals:
 //
 // #include <string_view>
 // using namespace std::string_view_literals;

